pkgs/top-level/python-packages.nix

```diff
@@ -27407,4 +27407,33 @@ in modules // {
      };
   };
 
+  Quandl = buildPythonPackage rec {
+     version = "3.0.0";
+     name = "Quandl-${version}";
+
+     src = pkgs.fetchurl {
+       url= "mirror://pypi/q/quandl/${name}.tar.gz";
+       sha256 = "d4e698eb39291e0b281975813054101f3dfb379dead10d34d7b536e1aad60584";
+     };
+
+     propagatedBuildInputs = with self; [
+	numpy
+	ndg-httpsclient
+	dateutil
+	inflection
```

Comment:
ok, if you have packages that depend on each other, then put them in one PR but separate commits